### ISSUE TITLE
Copilot: Tab to Accept Suggestion Not Inserting

### ISSUE THREAD

**Issue by GGabc** [LunarVim/LunarVim#3808]
### Problem description

Trying to use Copilot to create Fortran subroutines in Visual Studio with OneAPI Intel Fortran.  Copilot window shows suggested code for a convert Unix epoch time to UTC.  The inline suggestion displays inline also.  Tab is supposed to insert the suggestion.  That only hides the inline suggestion.  
Under menu>Tools>Options>Envir>Keyboard it shows that Edit.Copilot.TriggerInlineSuggestion having a Global shortcut of Cntl-Tab. 
The pull-down menu Edit>Copilot > Suggestions says to use keyboard shortcut cntl-shft-Tab to Trigger inline suggestions.  Is that to display the suggestions or to actual insert/apply the suggestion?  
>>> How to apply the inline suggestion.  Hitting tab or the pop-up tab displayed only erases the suggestion. 

In the Copilot panel, pressing the "Accept Solution" button only closes the Copilot panel without inserting anything.  Reopening the Copilot panel displays the endlessly spinning Copilot.  

I turned off all the extensions except Copilot and the Intel Fortran extensions.

Microsoft Visual Studio Community 2022
Version 17.4.4
VisualStudio.17.Release/17.4.4+33213.308
Microsoft .NET Framework
Version 4.8.04084

Installed Version: Community

Visual C++ 2022   00482-90000-00000-AA368
Microsoft Visual C++ 2022

ASP.NET and Web Tools   17.4.326.54890
ASP.NET and Web Tools

Azure App Service Tools v3.0.0   17.4.326.54890
Azure App Service Tools v3.0.0

C# Tools   4.4.0-6.22608.27+af1e46ad38d900023f8b1a2839484e471ece1502
C# components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Common Azure Tools   1.10
Provides common services for use by Azure Mobile Services and Microsoft Azure Tools.

Cookiecutter   17.0.22263.6
Provides tools for finding, instantiating and customizing templates in cookiecutter format.

GitHub Copilot   1.71.0.0 (v1.71.0.0@2d84b3d4a)
GitHub Copilot is an AI pair programmer that helps you write code faster and with less work.

GitHub Copilot Agent   1.71.8269 (v1.71.0)

Intel® Fortran Compiler   Package ID: w_oneAPI_2023.0.0.25565
Intel® Fortran Compiler – toolkit version: 2023.0.0, extension version 23.0.0070.17, Package ID: w_oneAPI_2023.0.0.25565, Copyright © 2002-2022 Intel Corporation. All rights reserved.
* Other names and brands may be claimed as the property of others.

Microsoft JVM Debugger   1.0
Provides support for connecting the Visual Studio debugger to JDWP compatible Java Virtual Machines

NuGet Package Manager   6.4.0
NuGet Package Manager in Visual Studio. For more information about NuGet, visit https://docs.nuget.org/

Python - Django support   17.0.22263.6
Provides templates and integration for the Django web framework.

Python with Pylance   17.0.22263.6
Provides IntelliSense, projects, templates, debugging, interactive windows, and other support for Python developers.

Syntax Visualizer   1.0
An extension for visualizing Roslyn SyntaxTrees.

TypeScript Tools   17.0.10921.2001
TypeScript Tools for Microsoft Visual Studio

Visual Basic Tools   4.4.0-6.22608.27+af1e46ad38d900023f8b1a2839484e471ece1502
Visual Basic components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Visual F# Tools   17.4.0-beta.22512.4+525d5109e389341bb90b144c24e2ad1ceec91e7b
Microsoft Visual F# Tools

Visual Studio Inline Suggestions   2.2.2361.27430
Ghost text API for Visual Studio inline suggestions

Visual Studio IntelliCode   2.2
AI-assisted development for Visual Studio.

### LunarVim version

No idea what this is.

### Neovim version (>= 0.8.0)

No idea what this is.

### Operating system/version

Windows 10.0.19041 Build 19041

### Affected language servers

Fortran

### Steps to reproduce

Fresh update of Visual Studio
New install of Github Copilot
Fresh update of Intel OneAPI Fortran

### Actual behavior

Copilot suggested code does not insert.

### Expected behavior

It displays code I would like to use both inline and in the Copilot window.  Numerous variations of key presses and clicking does not insert any code.

### support info

No idea what this is requesting

**Comment by GGabc** [comment]
Somehow Copilot actually posted the suggestion into the VS editor window.  I was hovering for a while over the pop-up button Tab to paste when I clicked the button.  The suggestion was pasted.

I tried a few more times as similar as I thought the first time.  No joy.  Restarting VS a couple of times had no benefit.

Site:https://docs.github.com/en/copilot/configuring-github-copilot/configuring-github-copilot-in-visual-studio-code
says that the Tab key should map to the function "editor.action.inlineSuggest.commit" to accept an inline suggestion.  Looking in VS (menu>tools>Envir>Keyboard) **there is no function by that name or similar.** 

And just now, VS hung for a bit and crashed.
